web/satellite/src/utils/constants/ProjectMemberSortEnum.ts

```diff
@@ -6,3 +6,8 @@ export enum ProjectMemberSortByEnum {
     EMAIL,
     CREATED_AT,
 }
+
+export enum ProjectMemberSortDirectionEnum {
```

Comment:
not related to project members, wrong place to store this file, not sort - order